src/Apps/Settings/Routes/MyCollection/Components/MyCollectionEmptyState.tsx

```diff
@@ -95,24 +107,32 @@ const MobileLayout: React.FC = () => {
           as={RouterLink}
           variant="primaryBlack"
           to="/my-collection/artworks/new"
-          onClick={() => trackAddCollectedArtwork()}
```

Comment:
this will be done in a different ticket: [the ticket](https://artsyproduct.atlassian.net/browse/CX-3185)